src/backend/distributed/sql/downgrades/citus--11.0-1--10.2-4.sql

```diff
@@ -364,3 +364,9 @@ RESET search_path;
 DROP VIEW IF EXISTS pg_catalog.citus_stat_activity;
 DROP FUNCTION IF EXISTS pg_catalog.citus_stat_activity;
 DROP FUNCTION IF EXISTS pg_catalog.run_command_on_all_nodes;
+
+DROP FUNCTION IF EXISTS pg_catalog.get_nodename_for_nodeid;
+DROP FUNCTION IF EXISTS pg_catalog.get_nodeport_for_nodeid;
+
+DROP FUNCTION IF EXISTS pg_catalog.get_nodeid_for_gpid;
+DROP FUNCTION IF EXISTS pg_catalog.get_pid_for_gpid;
```

Comment:
No need for `IF EXISTS`, it can actually hide typos. Also, these functions should have their argument types listed.

Reply:
Added argument types.

I guess our upgrade tests would reveal typos.
I thought if for some reason these functions are dropped (which could also be the reason they are trying to downgrade) this would prevent getting errors.

Reply:
AFAIK users cannot drop these functions, the citus extension depends on them. It's worth trying out though, maybe I'm mistaken.

Reply:
> I guess our upgrade tests would reveal typos.

I guess we generally catch those via https://github.com/citusdata/citus/blob/master/src/test/regress/expected/multi_extension.out#L988

But still, following Jelte's comment make it stricter. I'm not sure we always have this test for all new versions.

> the citus extension depends on them.
```SQL
drop function pg_catalog.citus_nodeid_for_gpid;
ERROR:  cannot drop function citus_nodeid_for_gpid(bigint) because extension citus requires it
HINT:  You can drop extension citus instead.
Time: 0.469 ms
```
that is true, and so it makes sense to avoid IF EXISTS as long as we have the parameters (to avoid dropping wrong functions)

Reply:
Ok I removed them